app/views/conversations/show.html.erb

```diff
@@ -1,15 +1,100 @@
-<div class="mx-auto md:w-2/3 w-full flex">
-  <div class="mx-auto">
-    <% if notice.present? %>
-      <p class="py-2 px-3 bg-green-50 mb-5 text-green-500 font-medium rounded-lg inline-block" id="notice"><%= notice %></p>
-    <% end %>
+<div class="flex flex-row h-screen">
```

Comment:
Since `flex-row` is the default, I don't think I've ever seen it unless it was part of some responsive UI pivot between column and row layouts. Why include it?

Reply:
Good catch. I’m so used to react native, which flips the default to flex-col so I’m still in the habit of being explicit on flex-row. :)